WifiManager/doc/WifiPlugin.md

```diff
@@ -928,7 +928,18 @@ WifiManager interface events:
 <a name="event.onWIFIStateChanged"></a>
 ## *onWIFIStateChanged <sup>event</sup>*
 
-Triggered when the Wifi state changes. See `getCurrentState` for a list of valid Wifi states.
+Triggered when the Wifi state changes. See `getCurrentState` for a list of valid Wifi states. This event is triggered during below methods call sequence.
+ 
+### Methods
+ 
+| Method | Description | 
+| :-------- | :-------- | 
+| `connect` | Connects the device with given SSID successfully and triggers this event with wifi state CONNECT |
+| `disconnect` | Disconnects the device successfully and triggers this event with wifi state DISCONNECT |
+| `clearSSID` | Clears the SSID successfully and triggers this event  with wifi state DISCONNECT |
+| `initiateWPSPairing` | Pair the device successfully and triggers this event with wifi state CONNECT |
+ 
+Also see: [connect](#method.connect), [disconnect](#method.disconnect), [clearSSID](#method.clearSSID), [initiateWPSPairing](#method.initiateWPSPairing).
```

Comment:
It is good to add a list of methods that can trigger the onWifiStateChange event. But there are probably more than the 4 methods listed above that result in this event being generated.